src/cli/index.ts

```diff
@@ -0,0 +1,248 @@
+          }
+
+          for (const sound of target.sounds) {
+            const filename = `${target.name}/sounds/${sound.name}.${sound.ext}`;
+            const asset = Buffer.from(sound.asset as ArrayBuffer);
+            fs.mkdirSync(resolve(fullOutputPath, filename, ".."), { recursive: true });
+            fs.writeFileSync(resolve(fullOutputPath, filename), asset);
+          }
+        }
+      });
+
+      process.stdout.write(chalk.green`\nProject exported to Leopard format.`);
+      process.stdout.write(chalk.gray` Files written to ${chalk.white(fullOutputPath)}\n\n`);
+      process.stdout.write(
+        chalkBox([
+          { str: "To preview the project, run:" },
+          { str: `$ ${chalk.white`cd ${output}`}`, length: 5 + output.length },
+          { str: `$ ${chalk.white`npx vite`}`, length: 10 }
+        ])
+      );
+      break;
+    }
+    case "leopard-zip": {
+      const leopard = await writeStep(`${chalk.bold("Converting")} project to ${chalk.white("Leopard")}.`, () => {
+        const leopard = project.toLeopard();
+        return leopard;
+      });
+
+      const fullOutputPath = resolve(process.cwd(), output);
+
+      await writeStep(`${chalk.bold("Exporting")} project to zip file ${chalk.white(fullOutputPath)}.`, () => {
+        // First, check if file name is already taken
+        try {
+          fs.accessSync(fullOutputPath);
+          throw new StepError("Output file already exists.");
+        } catch (err) {
+          if (err instanceof Object && "code" in err && err.code === "ENOENT") {
+            // File does not exist, good
+          } else {
+            throw err;
+          }
+        }
+
+        const zip = new JSZip();
+
+        for (const [filename, contents] of Object.entries(leopard)) {
+          zip.file(filename, contents);
+        }
+
+        for (const target of [project.stage, ...project.sprites]) {
+          for (const costume of target.costumes) {
+            const filename = `${target.name}/costumes/${costume.name}.${costume.ext}`;
+            const asset = Buffer.from(costume.asset as ArrayBuffer);
+            zip.file(filename, asset);
+          }
+
+          for (const sound of target.sounds) {
+            const filename = `${target.name}/sounds/${sound.name}.${sound.ext}`;
+            const asset = Buffer.from(sound.asset as ArrayBuffer);
+            zip.file(filename, asset);
+          }
+        }
+
+        zip.generateNodeStream({ type: "nodebuffer", streamFiles: true }).pipe(fs.createWriteStream(fullOutputPath));
+      });
+
+      break;
+    }
+  }
+}
+
+function chalkBox(lines: { str: string; length?: number }[]): string {
+  const lengths = lines.map(line => line.length ?? line.str.length);
+  const boxInnerWidth = Math.max(...lengths) + 2;
+
+  let outputStr = "";
+  outputStr += chalk.bold.blue`╔${"═".repeat(boxInnerWidth)}╗`;
+  for (const line of lines) {
+    const length = line.length ?? line.str.length;
```

Comment:
`chalkBox` is beautiful, thank you.

Can we use [`wcwidth`](https://www.npmjs.com/package/wcwidth) here instead of providing `length`, though? It's a ridiculously popular package for exactly this purpose, and is [an implementation of an IEEE spec](https://github.com/timoxley/wcwidth/blob/master/docs/index.md).